components/__tests__/SectionTextButtonImage.test.tsx

```diff
@@ -0,0 +1,57 @@
+import "@testing-library/jest-dom/jest-globals";
+import "@testing-library/jest-dom";
+
+import { render, screen } from "@testing-library/react";
+
+import SectionTextButtonImage from "../SectionTextButtonImage/SectionTextButtonImage";
+
+describe(SectionTextButtonImage.name, () => {
+  it("renders a heading, a button and an image on bright background", () => {
+    render(
+      <SectionTextButtonImage
+        headline={"hallo"}
+        button={"about"}
+        image={{
```

Comment:
a great pattern is to call this imageProps for example because you want to override the imageProps of the image component here. this makes the wrapper component very flexible. imagine for the button for example maybe you want to add an onClick listener in this case. now you already need two props like button and buttonOnClick. it is better to do it like: buttonProps={{ children: "about", onClick: () => {...}}}.
also you would make the relation between the parents props and the childrens prop explicit.

Reply:
Sounds great, for further information I'm going to contact you.